aiohttp/web.py

```diff
@@ -318,10 +319,18 @@ def __repr__(self):
         return "<Application 0x{:x}>".format(id(self))
 
 
+class TerminateSignal(SystemExit):
+    code = 143
+
+
+def sigterm_handler(signum, frame):
+    raise TerminateSignal()
+
+
 def run_app(app, *, host=None, port=None, path=None, sock=None,
             shutdown_timeout=60.0, ssl_context=None,
             print=print, backlog=128, access_log_format=None,
-            access_log=access_logger, loop=None):
+            access_log=access_logger, handle_signals=True, loop=None):
```

Comment:
I actually answered later in the PR comments. It's not really "out of the box" for the event loop, it's out of the box for default synchronous usage of Python.